crates/astria-sequencer/src/app/mod.rs

```diff
@@ -1325,3 +1330,15 @@ struct PostTransactionExecutionResult {
     validator_updates: Vec<tendermint::validator::Update>,
     consensus_param_updates: Option<tendermint::consensus::Params>,
 }
+
+fn index_all_send_packet_event_attributes(events: &mut [Event]) {
```

Comment:
Should we make this function more broad? Maybe take a list of events we want to index, or a way to index everything?

Reply:
Code would be simple to index everything, just wasn't sure how proper/useful that would be. Happy to do this though

Reply:
Done in 94416b1.